netbox/data_source_netbox_vlan_group.go

```diff
@@ -14,27 +14,33 @@ func dataSourceNetboxVlanGroup() *schema.Resource {
 		Read:        dataSourceNetboxVlanGroupRead,
 		Description: `:meta:subcategory:IP Address Management (IPAM):`,
 		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     		schema.TypeString,
+				Computed: 		true,
+				Optional: 		true,
+				AtLeastOneOf: []string{"id", "name", "slug", "scope_type"},
+			},
 			"name": {
 				Type:         schema.TypeString,
 				Computed:     true,
 				Optional:     true,
-				AtLeastOneOf: []string{"name", "slug", "scope_type"},
+				AtLeastOneOf: []string{"id", "name", "slug", "scope_type"},
 			},
 			"slug": {
 				Type:         schema.TypeString,
 				Optional:     true,
 				Computed:     true,
-				AtLeastOneOf: []string{"name", "slug", "scope_type"},
+				AtLeastOneOf: []string{"id", "name", "slug", "scope_type"},
 			},
 			"scope_type": {
 				Type:         schema.TypeString,
 				Optional:     true,
 				ValidateFunc: validation.StringInSlice(resourceNetboxVlanGroupScopeTypeOptions, false),
 				Description:  buildValidValueDescription(resourceNetboxVlanGroupScopeTypeOptions),
-				AtLeastOneOf: []string{"name", "slug", "scope_type"},
+				AtLeastOneOf: []string{"id", "name", "slug", "scope_type"},
 			},
 			"scope_id": {
-				Type:         schema.TypeInt,
+				Type:         schema.TypeString,
```

Comment:
Reasoning: The api also expects a string: 
https://github.com/fbreckle/go-netbox/blob/master/netbox/client/ipam/ipam_vlan_groups_list_parameters.go#L273